joomla/joomla-cms
# [4.0] [com_banners] Rewrite jQuery in vanilla JS

### Summary of Changes

This PR moves the inline JS from com_banners to a separate file and has been rewritten in vanilla JS


### Testing Instructions

- Apply PR
- Go to Components >> Banners >> New (or open existing)
- Switch the `type` between `Image` and `Custom`

**When `Image` is selected, the following fields should be visible below:**

> Image, Width, Height, Alt Text

**When `Custom` is selected, the following fields should be visible below:**

> Custom Code

So exactly the same behaviour as before.
